osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -157,5 +158,8 @@ public enum GlobalAction
 
         [Description("Home")]
         Home,
+
+        [Description("Toggle notifications")]
+        ToggleNotifications
```

Comment:
This should reside right after `ToggleDirect` above, to respect the order given to this enum.

Reply:
I actually had it there, but in that order the hotkeys will switch allocations. Basically it won't migrate well from an older version. So I guess having it down there is messing with the logical order but it can't be helped considering how the values are stored.

Reply:
I would implement some kind of migration if the project has this kind of system in place. Just give me a hint on how to do it and I will take care of it.

Reply:
@vntxx is correct, this has to be last for now to avoid breaking existing keybindings.

We will eventually be moving away from storing enums as integer values in the database but that change will be done globally and is out of scope here.

Reply:
Yeah I see, my bad not knowing about that.